### PR TITLE
Add Dialogue to PromptCMD

### PR DESCRIPTION
**Pull request by CaenJones**
When I attempted to use "prompt" in the past, it hid my username so I could only see it after I sent a message.

### CODE REVIEW

**Comment by CaenJones** [comment]
YEAH!

**Comment by quackduck** [comment]
Afaict, the change here is that empty prompts aren’t allowed and print an error message. I think empty prompts should be allowed but we can have devbot DM something like “(Your prompt is now empty. Did you mean to get more info about your prompt? Run `man prompt` for more info)”. Also we don’t really need a whole new setPrompt function: just change the original one to print a message! We dont need the errors package either :)

File: commands.go [hunk 1]
```diff
@@ -488,8 +488,14 @@ func nickCMD(line string, u *User) {
 }
 
 func promptCMD(line string, u *User) {
-	u.Prompt = line
-	u.formatPrompt()
+    u.Prompt = line
+    u.formatPrompt()
+
+    if line == "" {
+        fmt.Println("(Your prompt is now empty. Did you mean to get more info about your prompt? Run man prompt for more info)")
```
Comment (quackduck) [inline commands.go:495]: You want to write to the user, not stdout!

File: commands.go [hunk 2]
```diff
@@ -488,8 +488,14 @@ func nickCMD(line string, u *User) {
...
+    } else {
```
Comment (quackduck) [inline commands.go:496]: I don't think we need a success message

**Review by quackduck — CHANGES_REQUESTED** [review]

File: commands.go [hunk 3]
```diff
@@ -488,8 +488,12 @@ func nickCMD(line string, u *User) {
...
 
 func promptCMD(line string, u *User) {
-	u.Prompt = line
-	u.formatPrompt()
+    u.Prompt = line
+    u.formatPrompt()
+
+    if line == "" {
+        MainRoom.broadcast(Devbot, "Your prompt is now empty. Did you mean to get more info about your prompt? Run man prompt for more info")
+    }
```
Comment (quackduck) [inline commands.go:496]:
```suggestion
	u.Prompt = line
	u.formatPrompt()
	if line == "" {
		u.writeln(Devbot, "(Your prompt is now empty. Did you mean to get more info about your prompt? Run `man prompt` for more info)")
	}
```

**Review by quackduck — APPROVED** [review]

**Pull request merged**